README.md

```diff
@@ -0,0 +1,85 @@
+# MITOC Website
+
+This repository contains the source code of https://mitoc.mit.edu. 
+
+All members of the MITOC community are welcome to suggest edits by opening pull requests!
+
+If you'd like to change anything on the website, you can:
+- Read the rest of this short document to figure out how things work
+- Suggest a change on a file by clicking on the "Edit this file" button (see screenshot below)
+- Don't hesitate to contact the MITOC webasters if you need any help!
+
+![Screen Shot 2019-11-16 at 8 16 03 PM](https://user-images.githubusercontent.com/11834997/69001621-fcb15300-08af-11ea-81cb-b698ac23825c.png)
+
+## General
+
+The website is built using [Jekyll](https://jekyllrb.com/), a static site generator.
+
+
+## Content edition
+
+### What to edit
+
+The website content is located in the following directories:
+- [`_pages`](/_pages) for pages
+- [`images`](/images) for images and pictures
+- [`docs`](/docs) for other static files (e.g. PDF documents)
+- [`_data`](/_data) for structured data used by the website. This includes:
+  - [Menu items](_data/menu.yml)
+  - Club [positions](_data/positions.yml) and [officers](_data/officers.yml)
+  - [Trip fees](_data/trip_fees.yml)
+  - [Gear rental prices and deposit](_data/gear_prices.yml)
+    - _Note that this file is not connected to the gear database, and that prices updates must separately be applied there._
```

Comment:
Actually now that you have CircleCI on here, we can add a step to the CircleCI process that will use the public gear query API I'm building out on the gear DB to auto-generate this page!